.travis.yml

```diff
@@ -27,3 +27,8 @@ script:
   - make test_ftp
   - make test_bioblend
   - make test_docker_in_docker
+
+# blocklist
+branches:
```

Comment:
Yes. I can't cancel the travis jobs. Is it because I am not admin in Backofenlab?